submissions/Zakkarat/memory-pair-game/app.js

```diff
@@ -0,0 +1,74 @@
+const cards = document.getElementsByClassName('flipper');
+const container = document.getElementsByClassName('container')[0];
+const modalWindow = document.getElementsByClassName('modal')[0];
+const modalContent = document.getElementsByClassName('modal-content')[0];
+const button = document.getElementsByTagName('button')[0];
+let founded = [];
+let images = ['champ1', 'champ2', 'champ3', 'champ4', 'champ5'];
+let check = [];
+let checkFirst = true;
+let checkSecond = true;
+let prevTarget = "/img/back.jpg";
+prepareArray();
+container.addEventListener('click', function(elem) {if(checkFirst && checkSecond && elem.target.src != prevTarget) cardFlip(elem)});
+
+button.addEventListener('click', function() {location.reload()});
+
+function cardsAdd(elem) {
+  container.innerHTML += `<div class="flip-container"><div class="flipper"><div class="front"><img src="./img/backCard.jpg" /></div><div class="back"><img src="./img/${elem}.jpg "</div></div></div>`
+}
+
+images.forEach(elem => cardsAdd(elem));
+function cardFlip(elem) {
+  let target = elem.target;
+  while (target != container) {
+  if ([...cards].some(card => card == target) && founded.every(elem => elem.src != target.lastElementChild.getElementsByTagName("img")[0].src)) {
```

Comment:
99.9% you need strict comparison not equality :)

Reply:
Understood. Changed everything now.